src/operations/create_collection.ts

```diff
@@ -41,6 +41,17 @@ export interface TimeSeriesCollectionOptions extends Document {
   granularity?: 'seconds' | 'minutes' | 'hours' | string;
 }
 
+/** @public
+ * Configuration options for clustered collections
+ * @see https://www.mongodb.com/docs/v5.3/core/clustered-collections/
+ */
+export interface ClusteredCollectionOptions extends Document {
+  name?: string;
+  key?: Document;
+  unique?: boolean;
+  v?: string;
```

Comment:
This is optional and if it is specified the only correct value is currently `v: '2'`.

Reply:
This should be an `int` - see https://docs.google.com/document/d/1hRnqe1pq7ErcguJ5Bl8si1kHGKlZg0h25qsa3roASuo/edit#bookmark=id.oh101ver26rd

Reply:
From that slack thread, it sounds like `v` should never be passed in by users.  So I think we should omit it from the public interface so that our interface matches the documentation for the `clusteredIndex` option [here](https://www.mongodb.com/docs/v5.3/reference/command/create/#std-label-create.clusteredIndex)